docs/graphql/manual/api-reference/schema-metadata-api/remote-relationships.rst

```diff
@@ -0,0 +1,228 @@
+     - true
+     - RemoteField_
+     - Joining parameters of the remote join
+
+.. _update_remote_relationship:
+
+update_remote_relationship
+--------------------------
+
+``update_remote_relationship`` is used to update an existing remote relationship.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+  {
+    "type": "update_remote_relationship",
+    "args": {
+      "name": "sample_remote_relationship",
+      "table": "users",
+      "hasura_fields": [],
+      "remote_schema": "my-remote-schema",
+      "remote_field": {
+        "messages": {
+          "arguments": {
+               "limit":10
+            }
+          }
+        }
+      }
+    }
+  }
+
+.. _update_remote_relationship_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - name
+     - true
+     - RemoteRelationshipName_
+     - Name of the remote relationship
+   * - table
+     - true
+     - :ref:`QualifiedTable <QualifiedTable>`
+     - Object with table name and schema
+   * - hasura_fields
+     - true
+     - [:ref:`PGColumn <PGColumn>`]
+     - List of PG columns in the table that will be used in the ``remote_field`` object.
+   * - remote_schema
+     - true
+     - :ref:`RemoteSchemaName <RemoteSchemaName>`
+     - Name of the remote schema to join with
+   * - remote_field
+     - true
+     - RemoteField_
+     - Joining parameters of the remote join
+
+.. _delete_remote_relationship:
+
+delete_remote_relationship
+--------------------------
+
+``delete_remote_relationship`` is used to delete an existing remote relationship.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+   {
+       "type" : "delete_remote_relationship",
+       "args" : {
+          "table":{
+             "name":"users",
+             "schema":"public"
+          },
+          "name":"sample_remote_relationship"
+       }
+   }
+
+.. _delete_remote_relationship_syntax:
+
+``delete_remote_relationship`` will delete an existing remote relationship.
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - table
+     - true
+     - :ref:`QualifiedTable <QualifiedTable>`
+     - Object with table name and schema
+   * - name
+     - true
+     - RemoteRelationshipName_
+     - Name of the remote relationship
+
+.. _RemoteRelationshipName:
+
+RemoteRelationshipName
+&&&&&&&&&&&&&&&&&&&&&&
+
+.. parsed-literal::
+
+  String
+
+
+RemoteField
+&&&&&&&&&&&
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+   {
+       "message": {
+          "arguments":{
+             "where": {
+                 "id":{
+                     "eq":"$id"
+                  }
+             }
+          }
+       }
+   }
+
```

Comment:
This can be the second example as it is more complicated. Maybe you can show that you can add a static value join as well:

```
   {
       "messages": {
          "arguments":{
             "where": {
                 "id": {
                   "gt":"$id"
                  },
                 "limit": 100
             }
          }
       }
   }
```

Indentation should be fixed